src/Jackalope/NodeType/NodeType.php

```diff
@@ -392,4 +393,28 @@ public function canRemoveProperty($propertyName, $throw = false)
 
         return true;
     }
+
+    /**
+     * @param $value
```

Comment:
`mixed`

please also add a short description to the method, like "Get a string representation of a value for error reporting."